### PR TITLE
v1.35.1

### PR DESCRIPTION
**Pull request by johnclary**
Moving these three PRs into Prod:
- https://github.com/cityofaustin/atd-moped/pull/1186
- https://github.com/cityofaustin/atd-moped/pull/1177
- https://github.com/cityofaustin/atd-moped/pull/1184

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by johnclary** [comment]
I will snap a backup of the RDS instance before merging. The Access DB migration is going to follow shortly after this.

**Review by mddilley — APPROVED** [review]
Gave the linked PRs some overdue 👀, and this all looks solid to me! Thanks for getting the patch together. 🚀

File: moped-etl/arcgis/README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ The AGOL layers can be found here:
 
 ### Get it running
 
-1. Configure an `env_file` according to the `env_template` example. You can find the AGOL username and password in the API Secrets vault in the team password store.
+1. Configure an `env_file` according to the `env_template` example. You can find the AGOL Scripts Publisher username and password in the API Secrets vault in the team password store.
 
-2. Create and activate a Python environment that meets the requirments in `reuirements.txt`. Alternatively, you can use the provided Dockerfile.
+2. Create and activate a Python environment that meets the requirments in `requirements.txt`. Alternatively, you can use the provided Dockerfile.
```
Comment (chiaberry) [inline moped-etl/arcgis/README.md:20]: missing another e in the first requirements

File: moped-database/migrations/1698428471744_create_signage_line_component/down.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-- delete component (will cascade to component_work_types)
+DELETE from public.moped_components WHERE
+        moped_components.component_name = 'Signage'
+        AND moped_components.component_subtype = 'Linear';
+
+-- reset coponent id sequence
```
Comment (chiaberry) [inline moped-database/migrations/1698428471744_create_signage_line_component/down.sql:6]: missing an m

File: moped-editor/package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
   "license": "CC0-1.0",
-  "version": "1.35.0",
+  "version": "1.35.1",
```
Comment (chiaberry) [inline moped-editor/package.json:5]: Wolf Run Point One!

**Review by chiaberry — APPROVED** [review]
run the wolf!

**Comment by johnclary** [comment]
thank you both! 🌔 🐺 🚢

**Pull request merged**